src/javascript/app/pages/user/account/settings/account_closure.js

```diff
@@ -370,15 +370,19 @@ const AccountClosure = (() => {
             } else {
                 $submit_loading.setVisibility(0);
                 $closure_container.setVisibility(0);
+                $('.client_logged_in, #client-logged-in, #topbar-msg').setVisibility(0);
                 $success_msg.setVisibility(1);
+                $('.client_logged_out').setVisibility(1);
+                $('#topbar').addClass('primary-bg-color-dark');
+                $('#menu-top').removeClass('smaller-font top-nav-menu');
+                $('#topbar').removeClass('secondary-bg-color');
                 $.scrollTo(0, 500);
 
                 sessionStorage.setItem('closingAccount', 1);
-                setTimeout(() => {
-                    // we need to clear all stored client data by performing a logout action and then redirect to home
-                    // otherwise it will think that client is still logged in and redirect to trading page
+
+                $('.close_main_modal').on('click', () => {
                     Client.sendLogoutRequest(false, Url.urlFor('home'));
-                }, 10000);
+                });
```

Comment:
this should be sufficient, we don't need to update the style above to fake log them out. if we change our styles later it will break the above hardcodes.